### PR TITLE
[Feature #158175314] Render Failed Jobs Tab

### PR DESCRIPTION
**Pull request by wasswa-derick**
#### What does this PR do?
* Renders an Authorized User's Unresolved Checks

#### Description of Task to be completed?
* Add tab to the Base User Page
* Create Page for Failing Checks ([Feature #158175314] Add Failed check tab and custom page)
* Retrieve all the Authorized User's Failing checks ([Feature #158175314] Render failed checks on the unresolved stuff tab)

#### How should this be manually tested?
- Clone the project from GitHub ```git clone https://github.com/andela/hc-wits-kla.git```
- After cloning, cd into the project directory and run the environment setup in the project README
- Run the project URL from the terminal console in your browser
- Create an Account and Login
- Add some checks, ping some of the checks, adjust the period and grace period to a lower time to have the check drop
- When the check drops, an alert is sent to the user and the check is added to the Failed Checks Tab

#### Any background context you want to provide?
- When the difference between the time now and the and a summation of the check's last ping, period and grace periods is positive then the check is down and an email alert is sent to the owner.

#### What are the relevant pivotal tracker stories?
* #158175314

#### Screenshots

MY CHECKS SCREENSHOT

screen shot 2018-06-27 at 16 28 03

MY FAILED CHECKS SCREENSHOT

screen shot 2018-06-27 at 16 28 23

### CODE REVIEW

**Comment by john555** [comment]
Add a screenshot in your PR.

File: templates/front/failed_checks.html [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+    <div class="modal-dialog">
+        <form id="update-name-form" class="form-horizontal" method="post">
+            {% csrf_token %}
+            <div class="modal-content">
+                <div class="modal-header">
+                    <button type="button" class="close" data-dismiss="modal">&times;</button>
+                    <h4 class="update-timeout-title">Name and Tags</h4>
+                </div>
+                <div class="modal-body">
+                        <div class="form-group">
```
Comment (john555) [inline templates/front/failed_checks.html]: Fix indentation.

**Review by john555 — CHANGES_REQUESTED** [review]

**Comment by john555** [comment]
Are there any dependencies that you forgot to add in the requirements.txt file? I am failing to run the app on your branch.

File: hc/front/tests/test_my_failed_checks.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+        # Desktop
+        self.assertContains(r, "icon-down")
+
+        # Mobile
+        self.assertContains(r, "label-danger")
+
+        self.assertContains(r, "Alices Failed Checks")
+
+        self.assertEqual("down", self.check.status)
```
Comment (ja-odur) [inline hc/front/tests/test_my_failed_checks.py:36]: add a new line at the end of this file

**Comment by Yiga-fred** [comment]
Good work @wasswa-derick  looking good.

**Review by john555 — APPROVED** [review]

**Pull request merged**